doc/manual/src/advanced-topics/distributed-builds.md

```diff
@@ -156,3 +156,19 @@ option `builders-use-substitutes` in your local `nix.conf`.
 
 To build only on remote builders and disable building on the local
 machine, you can use the option `--max-jobs 0`.
+
+## Troubleshooting
+
+### ssh
+
+even if the store is reachable with `nix store ping --store your_store_url`, the nix-daemon might not be able to reach it. nix-daemon operates as root so you need to make sure that the machine is reachable via ssh as root user.
+
+- add the `-vvv` flags to show more information on the potential problem.
+- make sure your store is reachable by the root user. `sudo ssh -i "your_ssh_key_path" your_user@your_hostname`
+- make sure your builders configuration contains the ssh_key, user, and hostName
+
+## Warnings
+
+### darwin
+
+When using nix-darwin resulting closure will be copied back to your local machine.
```

Comment:
it was unintuitive to me that using a remote builder would copy things back onto your machine.
If you are on darwin and you are building something on nixos, the closures are not valid for your system. I don't understand why it would try to copy back into your machine. Since the behavior was surprising to me, I tried to add a warning.

Reply:
That could be described in the general introduction of remote builds, as it's not specific to nix-darwin

Reply:
Agreed, would you think the fix is just removing this section here then ?
I'm happy to, just want to make sure I'm not misunderstanding.